internal.nix

```diff
@@ -172,14 +296,15 @@ rec {
 
         postPatch = ''
           ln -sf ${patchedLockfile packageLockJson} package-lock.json
+          ln -sf ${patchedPackagefile packageJson} package.json
         '';
 
         buildPhase = ''
           runHook preBuild
           mkdir -p node_modules/.hooks
-          declare -pf > $TMP/preinstall-env
           ln -s ${preinstall_node_modules}/node_modules/.hooks/prepare node_modules/.hooks/preinstall
-          npm install --offline --nodedir=${nodeSource nodejs}
+          export HOME=.
+          npm i --offline --nodedir=${nodeSource nodejs}
```

Comment:
I think `npm install` is better in this case. The shortcuts should be used on the CLI only. For code (that is mostly read) it should be the "verbose" version.

Reply:
sure, fine by me